src/game_logic/entity_configuration.ipp

```diff
@@ -335,80 +335,80 @@ ActorID scoreNumberActor(const ScoreNumberType type) {
     int(ScoreNumberType::S5000) == 3 &&
     int(ScoreNumberType::S10000) == 4);
 
-  const auto intType = static_cast<ActorID>(type);
-  return 123 + intType;
+  const auto intType = static_cast<int>(type);
+  return static_cast<ActorID>(static_cast<int>(ActorID::Score_number_FX_100) + intType);
```

Comment:
From my point of view these multiple casts are "hacky", but couldn't think of a better solution so far.

Reply:
Yeah agreed, we could do an explicit switch instead? A bit more code, but easier to read?